getBagsFunction.php

```diff
@@ -0,0 +1,13 @@
+<?php
+    /**
+         * fetches all bags
+         * 
+         * @return
+         */
+
+        function getBags(&$database) {
+        $query=$database->prepare("SELECT `brand`,`image`,`model`,`price`,`materials` FROM `bags`;");
+        $query->execute();
+        $result = $query->fetchAll();
+        return $result;
+        }
```

Comment:
Check indents, the function shouldn't be indented as it isn't nested in anything, and the functions body should be indented once.